sd-finetuner-workflow/sd-finetuner/requirements.txt

```diff
@@ -1,4 +1,4 @@
-diffusers>=0.9.0
+git+https://github.com/huggingface/diffusers/
```

Comment:
This should really be pinned to a hash, instead of accepting whatever master is.

Reply:
Also, you should be teliing it what the egg name will be.

i.e. the following example shows using a checkpoint + egg: `git+https://git@github.com/Dango233/diffusers.git@dango.dev.nov_update#egg=diffusers`

Reply:
Got it pointing to this commit which implements v-prediction: https://github.com/huggingface/diffusers/commit/6c56f05097f7d3c561f02dc1c27e3dd7e9f88ce1